test/input_gen/genInput.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+##
+# Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+#
+# SPDX-License-Identifier: Apache-2.0-only
+#
+# @file genTestInput.py
+# @brief Generate test input
+# @author Jijoong Moon <jijoong.moon@samsung.com>
+
+import sys
+import os
+import numpy
+import struct
+
+def gen_input(input_filename, batch, channel, height, width):
+    if os.path.isfile(input_filename):
+        os.remove(input_filename)
+        
+    for i in range(0,batch):
+        imarray=numpy.random.rand(channel, height, width)*255
+        with open(input_filename, 'ab') as outfile:
```

Comment:
`input_filename` vs. `outfile`? I think the terminology may cause some confusion.

Reply:
fixed. Thanks.